ktor-swagger/src/main/kotlin/de/nielsfalk/ktor/swagger/SwaggerSupport.kt

```diff
@@ -194,6 +194,10 @@ private abstract class BaseWithVariation<B : CommonBase>(
                 if (typeInfo.type != Unit::class) {
                     addDefinition(typeInfo)
                 }
+                listOf("application/json")
+            }
+            is BodyFromString -> {
+                listOf("text/plain")
```

Comment:
You don't actually do anything with these list unless I'm not understanding something.